$NetBSD: patch-Makefile,v 1.3 2024/03/16 22:25:06 nia Exp $

* Honour LDFLAGS
* Let pkgsrc pull in the correct curses library
* Honor pkgsrc man directory

--- Makefile.orig	2018-10-03 22:04:48.000000000 +0000
+++ Makefile
@@ -27,7 +27,6 @@ DESTDIR =
 PREFIX  ?= /usr/local
 
 CURSES  = ncursesw
-LDFLAGS ?= -s
 
 ifeq (Windows_NT,$(OS))
 	ifeq (,$(findstring CYGWIN,$(UNAME_S)))
@@ -35,16 +34,7 @@ ifeq (Windows_NT,$(OS))
 	endif
 endif
 
-ifeq ($(UNAME_S),Darwin)
-	CURSES := ncurses
-	LDFLAGS :=
-endif
-
-ifeq ($(DEBUG),1)
-	LDFLAGS :=
-endif
-
-LDLIBS   = -l$(CURSES)
+LDLIBS   = -lcurses
 
 all: $(TARGET)
 
@@ -61,8 +51,8 @@ clean:
 install:
 	install -d $(DESTDIR)$(PREFIX)/bin
 	install -m 755 $(TARGET) $(DESTDIR)$(PREFIX)/bin/$(TARGET)
-	install -d $(DESTDIR)$(PREFIX)/share/man/man1
-	install -m 644 mdp.1 $(DESTDIR)$(PREFIX)/share/man/man1/$(TARGET).1
+	install -d $(DESTDIR)$(PREFIX)/${PKGMANDIR}/man1
+	install -m 644 mdp.1 $(DESTDIR)$(PREFIX)/${PKGMANDIR}/man1/$(TARGET).1
 
 uninstall:
 	$(RM) $(DESTDIR)$(PREFIX)/$(TARGET)
